mapreads/Dockerfile

```diff
@@ -1,7 +1,8 @@
-FROM debian:8.6
+FROM maxulysse/samtools:1.1
+# FROM maxulysse/samtools:dev
 
 LABEL author="Maxime Garcia" \
-description="BWA 0.7.8 and SAMTools 1.3 image for use in CAW" \
+description="BWA 0.7.8 and SAMTools 1.4 image for use in CAW" \
```

Comment:
Though on long term we will use samtools 1.3, it is not available yet on UPPMAX nodes. 
Also, we are going to use bwa 0.7.15 (in fact we are already using it on some clusters) for ALT contigs